tiatoolbox/dataloader/wsireader.py

```diff
@@ -137,5 +144,166 @@ def read_region(self, start_w, start_h, end_w, end_h, level=0):
         im_region = openslide_obj.read_region(
             [start_w, start_h], level, [end_w - start_w, end_h - start_h]
         )
-        im_region = np.asarray(im_region)
+        im_region = np.asarray(im_region)[:, :, 0:3]
```

Comment:
I have opted to convert to RGB instead in this commit https://github.com/TIA-Lab/tiatoolbox/commit/91aca39790a731f003b257999498a6da2ceefceb as I thought it would be a better option. Please let me know if you think otherwise.

Reply:
Update with background composite in this commit https://github.com/TIA-Lab/tiatoolbox/pull/12/commits/3ca57ef9e7c81d399f730c3c604d1b7d1faae1cc